pkgs/development/java-modules/jna/default.nix

```
@@ -31,7 +31,7 @@ stdenv.mkDerivation rec {
     description = "Java Native Access";
     license = with licenses; [ lgpl21 asl20 ];
     maintainers = with maintainers; [ nagy ];
-    platforms = platforms.linux;
+    platforms = platforms.linux ++ [ "x86_64-darwin" "aarch64-darwin" "i686-darwin" ];
```

```suggestion
    platforms = platforms.linux ++ platforms.darwin;
```

Only extra value would be armv7